frontend/src/api/UserManagementApi.js

```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
 import { getAuthorizationHeader } from "./AuthenticationUtil";
 
+/* eslint-disable import/prefer-default-export */
```

Comment:
can we just make this a default export and refactor? is that really so much of a pain? (maybe in a separate PR addressing code smells, cause this change has nothing to do with the current PR)

Reply:
0a93243